spring-cloud-dataflow-docs/src/main/asciidoc/configuration.adoc

```diff
@@ -89,11 +89,24 @@ You can secure your REST endpoints, as well as the Data Flow Dashboard by enabli
 and requiring clients to authenticate using either:
 
 * http://oauth.net/2/[OAuth 2.0]
-* Basic Authentication
+* Traditional Authentication (Incl. Basic Authentication)
+
+.Authentication Options
```

Comment:
I'd also suggest explaining the mutual exclusivity of auth options. What happens when multiple options were enabled? What takes precedence would be very useful.